src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -583,15 +583,20 @@
       usage =
           "Sets what language spec that input sources conform. "
               + "Options: ECMASCRIPT3, ECMASCRIPT5, ECMASCRIPT5_STRICT, "
-              + "ECMASCRIPT6 (default), ECMASCRIPT6_STRICT, ECMASCRIPT6_TYPED (experimental)"
+              + "ECMASCRIPT6_TYPED (experimental), ECMASCRIPT_2015, ECMASCRIPT_2016, "
+              + "ECMASCRIPT_NEXT"
     )
-    private String languageIn = "ECMASCRIPT6";
+    private String languageIn = "ECMASCRIPT_NEXT";
```

Comment:
`ECMASCRIPT_NEXT` would include features that aren't part of the current standard yet, so we shouldn't make it the default.
This could be `ECMASCRIPT_2016`, or if you like you could introduce an alias `ECMASCRIPT_CURRENT` that we'll update as new versions of the standard are released.